CMakeLists.txt

```diff
@@ -24,8 +24,6 @@ if(NOT SWIFT_SYSTEM_NAME)
   endif()
 endif()
 
-include(PlatformInfo)
```

Comment:
It doesn't look like the other patch to add the `include(PlatformInfo)` landed on this branch so we're not running the `PlatformInfo` code at all and thus we don't have a swiftmodule triple set. We can drop this commit.